src/coreclr/jit/compiler.h

```diff
@@ -4512,7 +4508,7 @@ class Compiler
 #endif // FEATURE_JIT_METHOD_PERF
 
     bool fgModified;             // True if the flow graph has been modified recently
-    bool fgComputePredsDone;     // Have we computed the bbPreds list
+    bool fgPredsComputed;        // Have we computed the bbPreds list
```

Comment:
Can you educate me why we need this at all? I assume it's `false` only during importing?

Reply:
We actually build pred lists before importation, so `fgPredsComputed` is true for all phases. So it seems like something we don't need. But I kept this concept around for two reasons:
* we sometimes do a bit of flow graph modification very early before we have added preds lists, and call methods that are also called after we have pred edges; those methods need to behave differently before and after (mainly `fgEnsureFirstBBisScratch` and friends
* we want to make sure no other pred-list sensitive method is called early.